GlobalBlueprint.cs

```diff
@@ -146,11 +160,143 @@ public void Trigger(Player player)
             }
         }
 
-        internal void Deconstruct(out string blueprintName, out BuildingType buildingType, out int materialCount)
+        internal void Deconstruct(out string blueprintName, out ResourceType buildingType, out int materialCount)
         {
             blueprintName = BlueprintName;
-            buildingType = BuildingType;
+            buildingType = ResourceType;
             materialCount = MaterialCount;
         }
     }
-}
+
+    public enum ResourceType
```

Comment:
Why you had to add that (seems like its exactly the same enum)...

If it dont have some values you have to commit new libs (in libs folder) from game folder.

Reply:
It's mostly the same but has any Building Type that is not a material removed (did that on the latest commit) since they don't do anything for triggers and also includes the 5 Experiment Resources

Unfortunately with this style we'll need to manually add any newly created buildings. I attempted to make this dynamic with BuildingTypes by manipulating the value assigned to TriggerBlueprintBuildingTypeColumn.DataSource, but I was not able to get this to work (exceptions thrown) and wasn't able to determine the underlying problem. 

If you know of a technique where can keep using BuildingType enum but add the 5 experiment resources into the drop down let me know and I can try to implement it

Reply:
Maybe you could extend that enum?
Im not sure if thats possible in .NET

Or just create list with all elements from that enum + extra 5 values?